start_provider.sh

```diff
@@ -2,7 +2,7 @@
 export PROVIDER_PRIVATE_KEY=0xfd5c1ccea015b6d663618850824154a3b3fb2882c46cefb05b9a93fea8c3d215
 export TEST_PRIVATE_KEY1=0xef4b441145c1d0f3b4bc6d61d29f5c6e502359481152f869247c7a4244d45209
 export TEST_PRIVATE_KEY2=0x5d75837394b078ce97bc289fa8d75e21000573520bfa7784a9d28ccaae602bf8
-export OPERATOR_SERVICE_URL=https://c2d-dev.operator.oceanprotocol.com/
+export OPERATOR_SERVICE_URL=http://172.15.0.13:31000
```

Comment:
why do we need this? Are we sure this is not used in production, like it was with start_aquarius.sh?

Reply:
I have updated the URLs. This is a script for development environment and that is the local URL for operator service, not
`https://c2d-dev.operator.oceanprotocol.com/`.